packages/e2e-tests/specs/experiments/customizing-widgets.test.js

```diff
@@ -562,6 +562,16 @@ async function addBlock( blockName ) {
 	);
 	await addBlockButton.click();
 
+	// TODO - remove this timeout when the test plugin for disabling CSS
+	// animations in tests works properly.
+	//
+	// This waits for the inserter panel animation to finish before
+	// attempting to insert a block. If the panel is still animating
+	// puppeteer can click on the wrong block.
+	//
+	// eslint-disable-next-line no-restricted-syntax
+	await page.waitForTimeout( 300 );
+
 	const blockOption = await find( {
 		role: 'option',
 		name: blockName,
```

Comment:
Can't we use normal selectors here? And then we can just wait for the selector?

Reply:
Ah, I guess that wouldn't work with the animation anyway.

Reply:
`find` does wait for the element, but yeah, the problem wasn't about that.